sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1801,7 +1801,7 @@ BRACKETED_EMPTY_COMMENT
     ;
 
 BRACKETED_COMMENT
-    : '/*' ~[+] .*? '*/' -> channel(HIDDEN)
+    : '/*' ~[+] ~'/'*? ( ~'*' '/' ~'*' )*? ~'*'+ BRACKETED_COMMENT? ~'*'+ ( ~'*' '/' ~'*' )*? ~'/'*? '*/' -> channel(HIDDEN)
```

Comment:
we may need some comments to explain what it does.

Reply:
OK